example/eblob_backend.cpp

```diff
@@ -357,27 +362,26 @@ int blob_file_info_new(eblob_backend_config *c, void *state, dnet_cmd *cmd, stru
 		}
 	}
 
-	auto response = serialize(dnet_lookup_response{
-		wc.flags,
-		ehdr.flags,
-		std::move(filename),
-
-		jhdr.timestamp,
-		wc.data_offset + json_offset,
-		json_size,
-		jhdr.capacity,
-		std::move(json_checksum),
+	std::unique_ptr<n2::lookup_response>
+		response(new n2::lookup_response(request->cmd,
+						 wc.flags, // record_flags
+						 ehdr.flags, // user_flags
+						 std::move(filename), // path
+						 jhdr.timestamp, // json_timestamp
+						 wc.data_offset + json_offset, // json_offset
+						 json_size, // json_size
+						 jhdr.capacity, // json_capacity
+						 std::move(json_checksum), // json_checksum
+						 ehdr.timestamp, // data_timestamp
+						 wc.data_offset + data_offset, // data_offset
+						 data_size, // data_size
+						 std::move(data_checksum))); // data_checksum
 
-		ehdr.timestamp,
-		wc.data_offset + data_offset,
-		data_size,
-		std::move(data_checksum),
-	});
-
-	err = dnet_send_reply(state, cmd, response.data(), response.size(), 0, context);
+	err = req_info->repliers.on_reply(std::move(response));
 	if (err) {
-		DNET_LOG_ERROR(c->blog, "{}: EBLOB: blob-file-info-new: dnet_send_reply: data: {:p}, size: {}: {} [{}]",
-		               dnet_dump_id(&cmd->id), response.data(), response.size(), strerror(-err), err);
+		DNET_LOG_ERROR(c->blog, "{}: EBLOB: blob-file-info-new: on_reply: "
+		                        "fd: {}, json_size: {}, data_size: {}: {} [{}]",
+		               dnet_dump_id(&cmd->id), wc.data_fd, json_size, data_size, strerror(-err), err);
```

Comment:
Why is `wc.data_fd` needed here?

Reply:
I totally didn't know what to log here, because previously here was logging of info about serialized buffer, but now we haven't such info. I think you're right, `wc.data_fd` is not relevant here.